File: Bronze l/1924.js
const input = require('fs').readFileSync('/dev/stdin').toString().split(' ');

// month 지정
const month = input[0];

//day 지정
const day = input[1];

//week 지정
const week = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];

// week 중에서 2007년의 요일을 계산하기
console.log(week[new Date(`2007-${month}-${day}`).getDay()]);

File: Bronze l/1924.py
days = [31,28,31,30,31,30,31,31,30,31,30,31]
weeks = ["MON","TUE","WED","THU","FRI","SAT","SUN"]
m,n = map(int,input().split(' '))
a = 1
b = 1
daysIndex = 0
index = 0
while True:
    if(a == m and b == n):
        print(weeks[index%7])
        break;
    
    if(b == days[daysIndex]):
        a += 1
        b = 0
        daysIndex+=1

    
    b+= 1
    index += 1